pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -518,10 +568,16 @@ func (exp *FlowExporter) addConnToSet(conn *flowexporter.Connection) error {
 		case "tcpState":
 			ie.SetStringValue(conn.TCPState)
 		case "flowType":
-			ie.SetUnsigned8Value(exp.findFlowType(*conn))
+			ie.SetUnsigned8Value(flowType)
 		}
 	}
-	err := exp.ipfixSet.AddRecord(eL, templateID)
+	//klog.InfoS("flow:", "flowtype", flowType, "field_length", len(eL))
+	filteredEL := exp.filterField(fieldFilterMap[flowType], eL)
+	if connectType == DeniedConnection {
+		filteredEL = exp.filterField(fieldFilterDeniedConnection, filteredEL)
+	}
+	//klog.InfoS("new_flow:", "flowtype", flowType, "connectType", connectType, "field_length", len(filteredEL))
```

Comment:
are lines 574 and 579 only for your debugging purposes, or should we un-comment them?